apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2

```diff
@@ -1696,14 +1696,14 @@
                 {% endfor %}
             {% endif %}
         ],
+        "isXacmlConnectorEnabled": "{{ console.ui.isXacmlConnectorEnabled }}",
```

Comment:
Shouldn't this be under `legacyMode`?

Reply:
Also, you may have to remove the `""` around the value and add a null check to avoid syntax errors.

```j2
{% if console.ui.isXacmlConnectorEnabled is defined %}
"isXacmlConnectorEnabled": {{ console.ui.isXacmlConnectorEnabled }},
{% endif %}
```

Reply:
`regardless of the legacy mode, the connector could be used` yes. It doesn't need to depend on the legacyMode config.